src/CommandLine/NServiceBus.Transport.AzureServiceBus.CommandLine.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net6.0</TargetFramework>
+    <TargetFrameworks>netcoreapp3.1;net6.0</TargetFrameworks>
```

Comment:
Did you mean to revert this?

Reply:
No I didn't. Somewhere along the line, a backup file got created for this and the related test project files somehow. I'm guessing this is a consequence of that.